### DISCUSSION TITLE
Projected range : issue or not ?

### DISCUSSION THREAD

**Discussion by pihomeserver** [teslamate-org/teslamate#1543] [General]
Hello
I have a Model 3 SR+ bought end of june 2020.
Using TM since i bought my car i have a constant decrease of my projected range and i don't understand why. I lost 15km in nearly 10 months. Can it be because of the Teslamate calculation, battery degradation and/or car usage ?

<img width="1372" alt="Projected range" src="https://user-images.githubusercontent.com/3817578/114844671-4cfd4f00-9ddb-11eb-9a60-48d65993d0eb.png">

Any feedback is welcome

Environment

TeslaMate version: latest
Type of installation: Docker
OS TeslaMate is installed on: Linux Debian
User OS & Browser: Mac Chrome/Brave

**Comment by ngardiner** [comment]
Batteries degrade up to 10% in the first year

**Reply by pihomeserver** [reply]
I saw that 10% is usually after 100k km.

**Reply by woogamer** [reply]
I have same problem just like you.
My car is M3 SR+ and my projected range is about 350km.
I only drove 9000km for a year and keep charging 20 to 80 in most times.
I asked tesla mechnic about my battery degradation. but he said my batter is degraded by 2%.
I am very confused about this.

**Reply by pihomeserver** [reply]
Not sure how to explain that to them
It seems that when weather is warmer the range increases a little bit
After 14000Km i have now a range estimated between 345 and 355 km

**Reply by pihomeserver** [reply]
*out of subject* Have to deep check but after some drives i have a loud sound of a fan running after leaving the car for 10-20 minutes. Even if outside temp is "normal" like 15° and after small drives (less than 10km). I checked via the app and climate control was not running. I guess that it impacts the battery and then remaining km ....

**Comment by ktsenvolt** [comment]
![image](https://user-images.githubusercontent.com/80360674/233846123-e9991d21-a682-49c3-9a19-8f344ec215aa.png)
This is my 2020 M3 SR+ after 2.5 years.

**Reply by pihomeserver** [reply]
Now my range is between 349-353 km. looks stable since 10 months
